src/core/lib/surface/call.cc

```diff
@@ -1475,9 +1362,8 @@ static void receiving_stream_ready(grpc_exec_ctx* exec_ctx, void* bctlp,
       grpc_byte_stream_destroy(exec_ctx, call->receiving_stream);
       call->receiving_stream = nullptr;
     }
-    add_batch_error(exec_ctx, bctl, GRPC_ERROR_REF(error), true);
-    cancel_with_error(exec_ctx, call, STATUS_FROM_SURFACE,
-                      GRPC_ERROR_REF(error));
+    bctl->rsr_error = GRPC_ERROR_REF(error);
```

Comment:
Why do we still need to track this?

Reply:
These errors are still needed for grpc_call_start_batch_and_execute()